deploy/template.yaml

```diff
@@ -589,8 +589,16 @@ Resources:
               Value: !GetAtt ApiGwHttpEndpoint.ApiEndpoint
             - Name: GTM_ID
               Value: !If [ IsProduction, "GTM-TT5HDKV", "GTM-TK92W68" ]
+            - Name: GTM_ID_GA4
+              Value: !If [ IsProduction, "GTM-K4PBJH3", "GTM-KD86CMZ" ]
             - Name: ANALYTICS_DOMAIN
-              Value: !If [ IsProduction, "account.gov.uk", !Sub "${Environment}.account.gov.uk" ]
+              Value: !If
+                - IsProduction
+                - "account.gov.uk"
+                - !If
+                  - IsNewDevelopment
+                  - !Sub "${Environment}.dev.identity.account.gov.uk"
+                  - !Sub "${Environment}.account.gov.uk"
```

Comment:
Currently this gets set to (something like) `dev-joee.account.gov.uk` in dev, but this isn't right - and explains why it's not properly saving cookie preferences.

The new analytics mechanism rechecks the preferences even after accepting, so without this change no analytics will work in dev.